app/presenters/vacancy_presenter.rb

```diff
@@ -24,11 +24,11 @@ def salary_range(del = 'to')
     range += "#{number_to_currency(model.maximum_salary)} "
     range += if model.pro_rata_salary?
                I18n.t('jobs.per_year_pro_rata')
-             elsif model.flexible_working?
-               I18n.t('jobs.per_year_fte')
-             else
-               I18n.t('jobs.per_year')
-             end
+    elsif model.flexible_working?
```

Comment:
this is strange, I don't think that is readable like this, you may want to change the settings to leave it the way it was before

Reply:
fixed this in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1095/commits/191ab8ea76356eabb8ff97e36f389f760a129525